specification/azurestackhci/AzureStackHCI.StackHCIVM.Management/main.tsp

```diff
@@ -16,8 +16,6 @@ import "./VirtualMachineInstance.tsp";
 import "./HybridIdentityMetadata.tsp";
 import "./AttestationStatus.tsp";
 import "./GuestAgent.tsp";
-import "../Operations.Management/models.tsp";
-
 using TypeSpec.Rest;
```

Comment:
This removes the operations API from your 2024-02-01-preview API version spec. Is that a) unintentional, b) fixing a spec bug, or c) a backend service change? Note: your namespace is onboarded to Azure Policy, so if the service is actually eliminating the operations API it will cause Azure Policy compliance to stop working. There could be other negative impacts as well.

Reply:
@shraddhasun @mentat9 we had a breaking change review for this. The operations.json API has been moved to a separate folder as it is a common API that is shared by 2 separate teams who contribute to the same namespace. It no longer needs to be imported here.